source/extensions/tracers/opentelemetry/http_trace_exporter.cc

```diff
@@ -0,0 +1,86 @@
+#include "http_trace_exporter.h"
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "source/common/common/logger.h"
+#include "source/common/protobuf/protobuf.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+OpenTelemetryHttpTraceExporter::OpenTelemetryHttpTraceExporter(
+    Upstream::ClusterManager& cluster_manager,
+    envoy::config::trace::v3::OpenTelemetryConfig::HttpConfig http_config,
+    OpenTelemetryTracerStats& tracing_stats)
+    : cluster_manager_(cluster_manager), http_config_(http_config), tracing_stats_(tracing_stats) {}
+
+bool OpenTelemetryHttpTraceExporter::log(const ExportTraceServiceRequest& request) {
+
+  std::string request_body;
+
+  const auto ok = request.SerializeToString(&request_body);
```

Comment:
@AlexanderEllis I removed the `HttpFormat` from the config. I understood your intention (I think) but the code is not at all prepared to export via protobuf/Json so simply having the config value didn't offer anything. So I thought of removing it for simplicity and IF we want to support that, it can be added later, together with the rest of refactor that will be needed. Let me know what you think.